Std/Data/List/Lemmas.lean

```diff
@@ -749,7 +759,11 @@ theorem getD_eq_get? : ∀ l n (a : α), getD l n a = (get? l n).getD a
   | _a::_, 0   => rfl
   | _a::l, n+1 => get!_eq_getD l n
 
-/-! ### take and drop -/
+/-! ### take -/
+
+@[simp]
+theorem take_cons (n) (a : α) (l : List α) : take (succ n) (a :: l) = a :: take n l :=
```

Comment:
This is essentially the same as the next theorem. There is a difference but the names should be swapped for when that distinction matters.

Reply:
I removed the `succ` variant.